doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -203,7 +203,7 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-|no_changes_yet_note|
+* Added support for the :ref:`nrf_desktop_watchdog` in the release configuration for the :ref:`zephyr:nrf54h20dk_nrf54h20`.
```

Comment:
`Added support` -> `Enabled`?

Reply:
In the changelog, we have this convention of having "subheadings" Added, Updated, Removed. Therefore, suggesting to keep this as is.

Reply:
So maybe:
```
Updated the :ref:`zephyr:nrf54h20dk_nrf54h20` release configuration to enable :ref:`nrf_desktop_watchdog`
```
We just enabled watchdog module for the nRF54H20 DK's release configuration here (no special action was needed on application's end to add watchdog support for nRF54H20 DK).

Reply:
Sounds good.